secrets/secrets-vault/charts/wasmcloud-secrets-vault/templates/service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "wasmcloud-secrets-vault.fullname" . }}
+  labels:
+    {{- include "wasmcloud-secrets-vault.labels" . | nindent 4 }}
+spec:
+  type: {{ .Values.service.type }}
+  ports:
+    - port: {{ .Values.service.port }}
+      targetPort: http
+      protocol: TCP
+      name: http
```

Comment:
Super nit: Might be worth using a more descriptive name to identify this is for the JWKS endpoint